Resources/Prototypes/_RMC14/Recipes/Construction/Graphs/structures/barricade_wood.yml

```diff
@@ -0,0 +1,29 @@
+- type: constructionGraph
+  parent: RMC
+  id: BarricadeWoodGraph
+  start: start
+  graph:
+  - node: start
+    actions:
+    - !type:DeleteEntity
+    edges:
+    - to: nodeWood
+      completed:
+      - !type:SnapToGrid
+      steps:
+      - material: RMCWood
+        amount: 5
+        doAfter: 3
+
+  - node: nodeWood
+    entity: RMCBarricadeWood
+    edges:
+    - to: start
+      completed:
+      - !type:SpawnPrototype
+        prototype: RMCPlankWood1
+        amount: 5
+      - !type:DeleteEntity
+      steps:
+      - tool: Prying
+        doAfter: 5
```

Comment:
It doesn't seem like you can deconstruct these in 13 so remove this unless you know otherwise